contracts/LiquidPledging.sol

```diff
@@ -400,21 +559,32 @@ contract LiquidPledging is LiquidPledgingBase {
         callPlugins(false, from, to, amount);
     }
 
-    // This function does 2 things, #1: it checks to make sure that the pledges are correct
-    // if the a pledged project has already been committed then it changes the owner
-    // to be the proposed project (Pledge that the UI will have to read the commit time and manually
-    // do what this function does to the pledge for the end user at the expiration of the commitTime)
-    // #2: It checks to make sure that if there has been a cancellation in the chain of projects,
-    // then it adjusts the pledge's owner appropriately.
-    // This call can be called from any body at any time on any pledge. In general it can be called
-    // to force the calls of the affected plugins, which also need to be predicted by the UI
+    /// @notice `normalizePledge` does 2 things:
+    ///   #1: Checks to make sure that the pledges are correct. Then if 
```

Comment:
"Checks to make sure that the pledges are correct" is a bit ambiguous.

maybe remove that and say something in effect of this method only affects pledges in the `Pledged` state

Reply:
```
    /// @notice If the pledge specified with idPledge is not pledged,
    ///  `normalizePledge` does 2 things:
    ///   #1: Changes the owner to be the proposed project (The UI 
    ///       will have to read the commit time and manually do what
    ///       this function does to the pledge for the end user
    ///       at the expiration of the commitTime)
```